.coveragerc

```diff
@@ -4,3 +4,4 @@ source =
     src/
 [report]
 fail_under = 72
```

Comment:
ok resolve conflicts and update coverage and we can merge it

Reply:
Done